package.json

```diff
@@ -10,6 +10,11 @@
     "lint": "eslint src example --ext .jsx --ext .js",
     "deploy-example": "gulp deploy-example"
   },
+  "files": [
+    "dist",
+    "src",
+    "example"
```

Comment:
Let's keep this in the next releases, as people may be reaching into the example folder to access the example SCSS file - #68